src/main/java/uk/ac/cam/cl/dtg/isaac/api/QuizFacade.java

```diff
@@ -683,7 +683,7 @@ public final Response markIncompleteQuizAttempt(@Context final HttpServletReques
                     "You can only mark assignments incomplete for groups you own or manage.").toResponse();
             }
 
-            if (assignment.getDueDate() != null && assignment.getDueDate().before(new Date())) {
+            if (assignment.getDueDate() != null && assignment.isBeforeDueDate(new Date())) {
```

Comment:
As far as I can tell, this line is causing the tests to fail. However, surely a test is "still due" if the current time is before the due date, i.e. the due date hasn't passed yet?